apps/dot_ext/loggers.py

```diff
@@ -67,7 +71,7 @@ def create_session_auth_flow_trace(request):
         # Get the application.
         Application = get_application_model()
```

Comment:
minor style preference, there are 4 lines to grab the app model manager: 
       # Get the application.
        Application = get_application_model()

I noticed some convention across BB2 code base, to put it outside of classes and on top of the source file...

Reply:
> minor style preference, there are 4 lines to grab the app model manager:
> # Get the application.
> Application = get_application_model()
> 
> I noticed some convention across BB2 code base, to put it outside of classes and on top of the source file...

@JFU-GIT That's right related to how it's used elsewhere. Also, it is being used in multiple functions so is best to be global here.  I just updated the code.  TY!

Reply:
@JFU-GIT I put this back. This logging related code gets run early and was getting an error with it global.